docs/imviz/plugins.rst

```diff
@@ -400,30 +396,12 @@ To import a regions file or object from the API:
   fp.import_region(region)
 
 
-
 .. _rotate-canvas:
 
 Canvas Rotation
 ===============
 
-.. note::
-
-    This plugin is deprecated in favor of rotation via :ref:`imviz-orientation` and will be removed
-    in a future release.
-
-The canvas rotation plugin allows rotating and horizontally flipping the image to any arbitrary
-value by rotating the canvas axes themselves.  Note that this does not affect the underlying data, and
-exporting data to the notebook via the API will therefore not exhibit the same rotation.
-
-The :ref:`imviz-compass` will also rotate (and flip) accordingly, but will show the zoom box
-corresponding to the zoom limits, not the region shown in the viewer itself.
-
-Presets are provided to reset the orientation as well as to set north up and east either to the
-right or the left, as well as a slider and input to set the angle and a switch to set whether the
-axes should be flipped horizontally after applying the rotation (a vertical flip can be achieved
-via a 180 deg rotation and a horizontal flip).
-
-Due to browser limitations, Canvas Rotation is only available on Chromium-based browsers.
+This plugin was removed in Jdaviz v4.0, use :ref:`imviz-orientation` instead.
```

Comment:
when (if ever) can we remove this section entirely (or maybe hide it somewhere but keep the link from breaking)?

Reply:
`.. _rotate-canvas:` needs a section to work, but it doesn't have to be on this page. And we can always break the doc anchor link in the future, or even in this PR if you don't think anyone is using it or it doesn't matter.

Reply:
maybe we can create a page dedicated to removed/deprecated stuff and migrate everything there?  I don't _suspect_ this was widely used, so it would be nice to clean it up and remove it from the main plugin page, but we can also revisit that later down the road if you think it should remain here for a little while.

Reply:
OK for simplicity, I just completely removed the section in this PR. If anyone complains about broken link, they can send in help call but like you said, probably no one. Can we merge now? Thanks!

Reply:
Yep, already approved